locale/templates/LC_MESSAGES/server.pot

```diff
@@ -538,10 +541,10 @@ msgstr ""
 #: lib/senders/templates/subscriptionDowngrade.txt:2
 msgid ""
```

Comment:
Should this have a new ID? Check this doc for the reason. https://github.com/mozilla-l10n/documentation/blob/main/src/localization/making_string_changes.md#why-is-it-necessary-to-use-new-ids